<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../pictures/appIcon.png</normaloff>../pictures/appIcon.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="inputPart" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#inputPart{
background-color:qlineargradient(spread:pad, x1:0.125, y1:0.238636, x2:1, y2:0.625, stop:0.448864 rgba(255, 216, 196, 255), stop:0.772727 rgba(190, 255, 178, 255));
}</string>
   </property>
   <widget class="QSpinBox" name="equationsInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
                border: 1.3px solid ;
                border-radius: 5px;
                padding: 4px;
                font-size: 15px;
			  background-color:#6e737a;
            }

            QSpinBox::up-button, QSpinBox::down-button {
                width: 11px;
                height: 11px;
            }

            QSpinBox::up-button {
                subcontrol-origin: border;
                subcontrol-position: top right;
            }

            QSpinBox::down-button {
                subcontrol-origin: border;
                subcontrol-position: bottom right;
            }</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 12pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter the number of equations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 12pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter number of max iterations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 12pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter the initial guesses</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="guessHolder" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-color: rgb(85, 83, 58);
border:2.2px solid;
</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QScrollArea" name="guessesArea">
       <property name="styleSheet">
        <string notr="true">#guessesArea{
background-color: rgb(255, 249, 217);
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="justGuesses">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>415</width>
          <height>155</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 12pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter the coefficients</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="maxIterationsInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
                border: 1.3px solid ;
                border-radius: 5px;
                padding: 4px;
                font-size: 15px;
			  background-color:#6e737a;
            }

            QSpinBox::up-button, QSpinBox::down-button {
                width: 11px;
                height: 11px;
            }

            QSpinBox::up-button {
                subcontrol-origin: border;
                subcontrol-position: top right;
            }

            QSpinBox::down-button {
                subcontrol-origin: border;
                subcontrol-position: bottom right;
            }</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="previousButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#previousButton{
	border: 1.3px solid #ccc;
	border-radius:5px; 
	font: 9pt &quot;MS Shell Dlg 2&quot;;
	
	background-color: rgb(90, 90, 90);
}

#previousButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Previous page</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../pictures/backArrow.png</normaloff>../pictures/backArrow.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 12pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter the tolerance value</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="toleranceInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
                border: 1.3px solid ;
                border-radius: 5px;
                padding: 4px;
                font-size: 10px;
			  background-color:#6e737a;
            }

            QDoubleSpinBox::up-button, QSpinBox::down-button {
                width: 11px;
                height: 11px;
            }

            QDoubleSpinBox::up-button {
                subcontrol-origin: border;
                subcontrol-position: top right;
            }

            QDoubleSpinBox::down-button {
                subcontrol-origin: border;
                subcontrol-position: bottom right;
            }</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>9</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#solveButton{
	border: 1.3px solid #ccc;
	border-radius:5px; 
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	
	background-color: rgb(90, 90, 90);
}

#solveButton:hover{
	background-color:#584082;
}
</string>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../pictures/calculator.png</normaloff>../pictures/calculator.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1.3px solid rgb(255, 255, 255); 
font: 10pt &quot;Arial&quot;;
background-color:rgb(181, 207, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Enter the number of significant figures</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="percisionInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
                border: 1.3px solid ;
                border-radius: 5px;
                padding: 4px;
                font-size: 15px;
			  background-color:#6e737a;
            }

            QSpinBox::up-button, QSpinBox::down-button {
                width: 11px;
                height: 11px;
            }

            QSpinBox::up-button {
                subcontrol-origin: border;
                subcontrol-position: top right;
            }

            QSpinBox::down-button {
                subcontrol-origin: border;
                subcontrol-position: bottom right;
            }</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>17</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>851</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 80, 26);
border-color: rgb(85, 85, 57);
border:2.2px solid;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="equationsArea">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 252, 219);
border-color: rgb(170, 137, 159);
border-size: 2px;</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>835</width>
         <height>255</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
